### PR TITLE
Fix rockcraft yaml

### PR DESCRIPTION
**Pull request by DanielArndt**
This builds on #9 . #9 is included to illustrate that the builds will pass when both are merged.

~~I'm actually unsure if the `services` section is even required. This is something I was told to add in the rock workshop, but admittedly I can't remember what difference it makes.~~

~~I can run the rock just fine without it, but~~ I need to add `exec` to the docker run command in either case (i.e. it doesn't work as shown in the README).

### CODE REVIEW

**Comment by gatici** [comment]
I agree that README.md needs to be updated. We need to add exec to the following command.
`docker run ghcr.io/canonical/lego:4.9.1 exec lego -d <your domain> -m <your email> --dns=<your dns provider> --accept-tos=true run`

File: rockcraft.yaml [hunk 1]
```diff
@@ -9,7 +9,11 @@ build-base: ubuntu:22.04
 platforms:
   amd64:
 
-cmd: ["lego"]
+services:
```
Comment (gruyaume) [inline rockcraft.yaml]: Related to your comment in the PR description.

The pebble services section is not needed. Especially in the context of charms where we will create our own pebble service. We can simply remove it here imo.

**Review by gruyaume — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yaml
Comment (gruyaume) [inline .github/workflows/build.yaml:1]: (Unrelated to this file or this PR in general)

We should add a CODEOWNERS file in this project to automatically assign @canonical/telco as a reviewer.

Comment (DanielArndt) [reply .github/workflows/build.yaml:1]: I'll cover this in another PR

File: rockcraft.yaml (hunk 1 above)
Comment (DanielArndt) [reply rockcraft.yaml]: Fixed in https://github.com/canonical/lego-rock/pull/10/commits/c588d2c71ae80d0f84e283be56dcd566422debfe

**Review by gruyaume — APPROVED** [review]

File: .github/workflows/build.yaml
Comment (DanielArndt) [reply .github/workflows/build.yaml:1]: https://github.com/canonical/lego-rock/pull/11

**Pull request merged**